src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/Legacy/TokenizerBackedParser.cs

```diff
@@ -705,11 +705,15 @@ protected void ConfigureSpanContext(SpanContextConfigAction? config)
 
     protected void ConfigureSpanContext(SpanContextConfigActionWithPreviousConfig? config)
     {
-        var prev = SpanContextConfig;
```

Comment:
Can this line be kept? It seems a bit harder to read when `SpanContextConfig` is set to an expression that uses the previous value of `SpanContentConfig`?